src/Interface/Modules/Visualization/CreateStandardColorMapDialog.cc

```diff
@@ -317,12 +302,11 @@ void AlphaFunctionManager::updateAlphaFunction()
 
   for (int i = 0; i < static_cast<int>(alphaFunction_.size()); ++i)
   {
-    if (i > 0 && i < alphaFunction_.size() - 1)
+    if (false && i > 0 && i < alphaFunction_.size() - 1)
```

Comment:
???

Reply:
haha whoops was testing to see if this code even mattered. Seems like it doesn't will take a closer look before removing entierly.